### DISCUSSION TITLE
"enabled" property doesn't work on a Tab of the TabWidget

### DISCUSSION THREAD

**Discussion by HitecExports** [slint-ui/slint#3995] [Q&A] [answered]
Hello!

Using Slint 1.3.0, Windows 11, wininit, femtovg / skia

```
import { TabWidget} from "std-widgets.slint";
export component Demo {
    width: 500px;
    height: 300px;
    TabWidget {
        Tab {
            enabled: false;                                     // NOT WORKING! 
            title: "First Tab";
            Text {
                text: "Content 1";
            }
        }
        Tab {
            title: "Second Tab";
            Text {
                text: "Content 2";
            }
        }
    }
}
```

**Property "enabled" doesn't work on a Tab of the TabWidget:**

![image](https://github.com/slint-ui/slint/assets/19405195/73d6a687-89ab-495f-9849-1f88ed636223)

**According to the documentation, this property should be supported:**

![image](https://github.com/slint-ui/slint/assets/19405195/3ffeb6ca-91e7-4a73-a719-0002be0c20be)

**Comment by ogoffart** [comment, marked as answer]
Thanks for writing a discussion.
The documentation is wrong and most of these properties are internal and don't exist.  Fixing in https://github.com/slint-ui/slint/pull/3998

It is currently not possible to disable Tab or TabWidget. That is currently not implemented.
